admin/class-h5p-plugin-admin.php

```diff
@@ -646,7 +664,7 @@ private static function asset_handle($path) {
    * @param string $path
    */
   public static function add_script($handle, $path) {
-    wp_enqueue_script(self::asset_handle($handle), plugins_url('h5p/' . $path), array(), H5P_Plugin::VERSION);
+    wp_enqueue_script(self::asset_handle($handle), plugins_url('h5p/' . $path), array(), H5P_Plugin::VERSION, true);
```

Comment:
Oh, I think this is not necessary at all, sorry for that. Is there any easy way of rejecting this from your side or do I have to update the pull request (sorry for my ignorance)?